pkg/status/status.go

```diff
@@ -125,6 +126,26 @@ func GetPlacementStatus(s *common.Status) common.PlacementStatusType {
 	return s.GetPlacementStatus().GetStatus()
 }
 
+// Set UploadError
+func SetUploadError(s *common.Status, err ...error) {
+	s.UploadStatus.LastUploadError = errors.ErrorToProto(err[0])
+}
+
+// SetUploadStatus
+func SetUploadStatus(s *common.Status, dProgressPercentage, dUploadSizeInBytes, dFileSizeInBytes int64, err ...error) {
```

Comment:
See above comment about export.

Out of scope for this PR - I'm curious why this file has `func Set...()` and passes in a `common.Status` - it'd make sense for the `Status` object to have methods to set its upload status, upload error, etc.

Reply:
Yeah I agree. This could be refactored to something like ```func (s *Status) SetUploadStatus()```. I am not sure why this is done this way though. I just followed the pattern I saw in the rest of the file.